integrationtest/src/main/java/software/amazon/timestream/integrationtest/DatabaseMetaDataOneDBMultiTBIntegrationTest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright <2020> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.timestream.integrationtest;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import software.amazon.timestream.jdbc.TimestreamDatabaseMetaData;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Integration tests for supported getters in {@link TimestreamDatabaseMetaData}
+ */
+class DatabaseMetaDataOneDBMultiTBIntegrationTest {
```

Comment:
I think we need a common test for the test framework. Then for each different region, we just need to fill input and expected result. Then we do not need to write so many duplicated test for each of them.

Reply:
I'm not understanding there, because input and expected result would be different, and the name patterns would be different too (we should not use the exact same name patterns for the tests). Databases/Tables would need to be created before every test, so it's simpler to have a test file in my mind.
Currently, the step could be: 
1. copy DatabaseMetaDataOneDBMultiTBIntegrationTest.java and rename to <newTest>.java
2. Enter region, database name, table name in Constants.java
3. Modify <newTest>.java based on test case. 
I think that's pretty simple so far. Please let me know how you think.